### ISSUE TITLE
#if !MIN_VERSION_base(4,7,0)

### ISSUE THREAD

**Issue by ekmett** [haskell-compat/base-orphans#4]
```haskell
#if !MIN_VERSION_base(4,7,0)
```

is used throughout the code, but this doesn't work on older `cabal` versions. because the macro expands without parentheses. You need

```haskell
#if !(MIN_VERSION_base(4,7,0))
```

or to only accept fairly recent versions of `cabal`, which is too restrictive.

**Comment by RyanGlScott** [comment]
Oops, I forgot about that issue. I'll try to get that fixed soon.

**Comment by RyanGlScott** [comment]
Fixed in 9302c8a1c6c39399cee868ec722f4e60e2aaf100.
